src/plugins/telemetry_management_section/public/components/telemetry_management_section.tsx

```diff
@@ -197,12 +201,9 @@ export class TelemetryManagementSection extends Component<Props, State> {
         />
       </p>
       <p>
-        <EuiLink onClick={this.toggleExample}>
-          <FormattedMessage
-            id="telemetry.seeExampleOfWhatWeCollectLinkText"
-            defaultMessage="See an example of what we collect"
-          />
-        </EuiLink>
+        See examples of the <EuiLink onClick={this.toggleExample}>cluster data</EuiLink> and{' '}
```

Comment:
We've lost the translations for "See examples of the" wording.
Suggestion:
```suggestion
       <FormattedMessage
        id="telemetry.seeExamplesOfWhatWeCollect"
        defaultMessage="See examples of the {clusterData} and {endpointSecurityData} that we collect."
        values={{
          clusterData: <EuiLink onClick={this.toggleExample}>cluster data</EuiLink>,
          endpointSecurityData: (
            <EuiLink onClick={this.toggleSecurityExample}>endpoint security data</EuiLink>
          ),
        }}
      />
```
or something along those lines

Reply:
This LGTM. Thanks!